src/TodoList.jsx

```diff
@@ -1,24 +1,12 @@
 import TodoListItem from "./TodoListItem";
 
-const todoList = [
-  {
-    id: 1,
-    title: "Review test cases for the new feature",
-  },
-  {
-    id: 2,
-    title: "Conduct regression testing on the latest build",
-  },
-  {
-    id: 3,
-    title: "Retest fixed bugs",
-  },
-];
-
-function TodoList() {
+function TodoList(props) {
```

Comment:
What's another way to destructure props here? Hint: It would be a change in line 3 that would result in being able to remove the code on line 5